src/compute-pr-actions.ts

```diff
@@ -487,12 +488,12 @@ function createWelcomeComment(info: ExtendedPrInfo, post: (c: Comments.Comment)
     display(``,
             `## Status`,
             ``,
-            ` * ${emoji(!info.hasMergeConflict)} No merge conflicts`);
+            ` * ${emoji.failure(!info.hasMergeConflict)} No merge conflicts`);
 
-    const expectedResults = info.ciResult === "unknown" ? "finished" : "passed";
-    display(` * ${emoji(info.ciResult === "pass")} Continuous integration tests have ${expectedResults}`);
+    const result = emoji.result(info.ciResult);
+    display(` * ${result.emoji} Continuous integration tests ${result.text}`);
```

Comment:
Maybe do this since `result` is a very generic name?

```suggestion
    { const result = emoji.result(info.ciResult);
      display(` * ${result.emoji} Continuous integration tests ${result.text}`); }
```

Reply:
This might be the first time in my career I've intentionally used a standalone block. 😄